src/Init/Data/BitVec/Bitblast.lean

```diff
@@ -1650,7 +1655,7 @@ theorem toInt_sdiv (a b : BitVec w) : (a.sdiv b).toInt = (a.toInt.tdiv b.toInt).
   · rw [← toInt_bmod_cancel]
     rw [BitVec.toInt_sdiv_of_ne_or_ne _ _ (by simpa only [Classical.not_and_iff_not_or_not] using h)]
 
-/-! ### Lemmas that use Bitblasting circuits -/
+/-! ### Lemmas that use Bit Blasting circuits -/
 
```

Comment:
The casing is off here. If you want to use `title case`, then `Circuits` should also be uppercase, no?

Reply:
I've restored the line in question to sentence case. The module docstrings aren't a part of the review process that I'm doing right now - I'm focusing on things that appear in hovers and in the reference manual at the moment.

Thanks!